README.md

```diff
@@ -8,6 +8,28 @@ A syntax tidy-upper (formatter) for PureScript.
 npm install -g purs-tidy
 ```
 
+## Usage
+
+You can use `purs-tidy` to format files in place or via STDIN / STDOUT (which is useful for editor integration):
+
+```sh
+# Formatting a collection of files in place:
+$ purs-tidy format-in-place "src/**/*.purs"
+
+# Using STDIN to format a file:
+$ purs-tidy format < MyFile.purs
+```
+
+### Configuration
+
+You can view the full configuration that `purs-tidy` accepts via flags here:
+https://github.com/natefaubion/purescript-tidy/blob/main/bin/Bin/FormatOptions.purs
```

Comment:
I'm not sure if this is super useful. Should we just reference `--help`?

Reply:
Yes! That's much better.